### PR TITLE
[16.0][MIG] sale_report_delivered_subtotal: Migration to version 16.0

### PR DESCRIPTION
**Pull request by Yadier-Tecnativa**
@Tecnativa TT42437

### CODE REVIEW

**Comment by Yadier-Tecnativa** [comment]
@sergio-teruel

**Review by sergio-teruel — APPROVED** [review]

**Comment by sergio-teruel** [comment]
@stefan-tecnativa Can you review? Thanks.

File: sale_report_delivered_subtotal/README.rst [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Usage
+=====
+
+Go to *Sales > Reporting > Sales* and you'll find *Subtotal Delivered* and *Gross Weight
```
Comment (CarlosRoca13) [inline sale_report_delivered_subtotal/README.rst:42]:
```suggestion
Go to *Sales > Reporting* and you'll find *Untaxed Total Delivered* and *Gross Weight
```

**Review by CarlosRoca13 — APPROVED** [review]
Just a minor comment

**Comment by pedrobaeza** [comment]
@Yadier-Tecnativa please perform the change

File: sale_report_delivered_subtotal/README.rst (hunk 1 above)
Comment (Yadier-Tecnativa) [reply sale_report_delivered_subtotal/README.rst:42]: Done, thanks for the review

Comment (pedrobaeza) [reply sale_report_delivered_subtotal/README.rst:42]: @Yadier-Tecnativa this change should be put on the `readme` folder, not here.

Comment (sergio-teruel) [reply sale_report_delivered_subtotal/README.rst:42]: Please @Yadier-Tecnativa Can you attend the comment

Comment (Yadier-Tecnativa) [reply sale_report_delivered_subtotal/README.rst:42]: Done in readme folder

**Comment by sergio-teruel** [comment]
ping @juangomezzurita

**Review by ernestotejeda — APPROVED** [review]
code and functionally tested

**Pull request merged**